### PR TITLE
Turn flashlight on at startup  bug resolved

### PR DESCRIPTION
**Pull request by im-indrajeet**
Turn flashlight on at startup setting is activate and after switching off flashlight user navigates away to other app and come back to this app again flashlight will be turned on. This regards to  issue #71

### CODE REVIEW

**Comment by tibbi** [comment]
fix some spacing around brackets and resolve the conflicts please

**Comment by im-indrajeet** [comment]
> fix some spacing around brackets and resolve the conflicts please

I resolved conflicts and fixed spacing around brackets

File: app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt [hunk 1]
```diff
@@ -146,11 +155,13 @@ class MainActivity : SimpleActivity() {
     }
 
     private fun launchSettings() {
+        reTurnFlashlightOn = false
         startActivity(Intent(applicationContext, SettingsActivity::class.java))
     }
 
     private fun launchAbout() {
-        val licenses = LICENSE_EVENT_BUS
```
Comment (im-indrajeet) [inline app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt:153]: There is two lines with same variable name val licenses = LICENSE_EVENT_BUS and val licenses = LICENSE_OTTO. Line val licenses = LICENSE_EVENT_BUS was not here previously I don't know which one to remove so i removed this line please look at this.

Comment (tibbi) [reply app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt:153]: keep EVENT_BUS only, that is apparently the newer one

Comment (im-indrajeet) [reply app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt:153]: > keep EVENT_BUS only, that is apparently the newer one

Done

**Comment by tibbi** [comment]
will test it later, not working on the Flashlight right now. Should be ok. Thanks

**Comment by im-indrajeet** [comment]
> will test it later, not working on the Flashlight right now. Should be ok. Thanks

Ok thanks

**Comment by tibbi** [comment]
looks good, hope it wont break anything :) Thanks

**Pull request merged**